llama_stack/core/server/metrics.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
How is this different from https://github.com/trallnag/prometheus-fastapi-instrumentator?